default_app/renderer.ts

```diff
@@ -27,8 +27,8 @@ function initialize () {
   }
 
   document.querySelector<HTMLAnchorElement>('.electron-version')!.innerText = `Electron v${process.versions.electron}`
-  document.querySelector<HTMLAnchorElement>('.chrome-version')!.innerText = `Chromium v${process.versions.chrome}`
-  document.querySelector<HTMLAnchorElement>('.node-version')!.innerText = `Node v${process.versions.node}`
+  document.querySelector<HTMLAnchorElement>('.chrome-version')!.innerText = `<a href="https://www.chromium.org/v${process.versions.chrome}">Chromium v${process.versions.chrome}</a>`
```

Comment:
You probably want `innerHTML` here, `innerText` would encode/decode the "text" you give it:

```suggestion
  document.querySelector<HTMLAnchorElement>('.chrome-version')!.innerHTML = `<a href="https://www.chromium.org/v${process.versions.chrome}">Chromium v${process.versions.chrome}</a>`
```

Reply:
You're right, the link doesn't work. I think that in cases like this we should just link to the general docs URL. Especially since I don't believe chrome has version-specific documentation.